Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -883,27 +884,30 @@ def get_mapping_fields_command():
     return mapping_response
 
 
-def get_modified_remote_data_command(client, args, xdr_delay: int = 0, last_mirroring: Optional[dict] = None):
+def get_modified_remote_data_command(client, args, xdr_delay: int = 1):
     remote_args = GetModifiedRemoteDataArgs(args)
-    last_update = last_mirroring.get(
-        'time') if isinstance(last_mirroring, dict) else remote_args.last_update
+    integration_context = demisto.getIntegrationContext()
+    last_update: str = integration_context.get('mirroring_last_update', remote_args.last_update)
     last_update_utc = dateparser.parse(last_update,
                                        settings={'TIMEZONE': 'UTC', 'RETURN_AS_TIMEZONE_AWARE': False})   # convert to utc format
 
     if last_update_utc:
         gte_modification_time_milliseconds = last_update_utc - timedelta(minutes=xdr_delay)
-        lte_modification_time_with_ms = gte_modification_time_milliseconds + timedelta(minutes=1)
+        lte_modification_time_milliseconds = gte_modification_time_milliseconds + timedelta(minutes=1)
     demisto.debug(
-        f'Performing get-modified-remote-data command. {last_update=} | {gte_modification_time_milliseconds=} |'
-        f'{lte_modification_time_with_ms=}'
+        f'Performing get-modified-remote-data command {last_update=} | {gte_modification_time_milliseconds=} |'
+        f'{lte_modification_time_milliseconds=}'
     )
     raw_incidents = client.get_incidents(
         gte_modification_time_milliseconds=gte_modification_time_milliseconds,
-        lte_modification_time_milliseconds=lte_modification_time_with_ms,
+        lte_modification_time_milliseconds=lte_modification_time_milliseconds,
         limit=100)
-    last_run_obj = demisto.getLastRun()
-    last_run_obj['mirroring'] = {'time': (lte_modification_time_with_ms + timedelta(milliseconds=1))}  # type: ignore
-    demisto.setLastRun(last_run_obj)
+
+    last_run_mirroring = (lte_modification_time_milliseconds + timedelta(milliseconds=1))
+    # Format with milliseconds as string, truncate microseconds
+    last_run_mirroring_str = last_run_mirroring.replace(tzinfo=pytz.UTC).strftime(
+        '%Y-%m-%d %H:%M:%S.%f')[:-3] + '+02:00'  # type: ignore
```

Comment:
Can you elaborate more on what those lines do and why do we need them? Why do we add 2? Are we sure that the first mirroring string is in UTC?